src/components/component.ts

```diff
@@ -263,8 +267,8 @@ module Plottable {
         this.anchor(selection);
       }
       if (this._element == null) {
-        throw new Error("If a Component has never been rendered before, then renderTo must be given a node to render to, \
-          or a d3.Selection, or a selector string");
+        throw new Error("If a Component has never been rendered before, then renderTo must be given a node to render to, " +
+          "or a d3.Selection, or a selector string");
```

Comment:
TBH, main reason was that my Sublime parser was dying here. However, this is better practice.

I generally dislike trailing `\` because misconfigurations in the editor might add an invisible whitespace right after the `\` and lead to errors. This format is more robust